seisbench/models/eqtransformer.py

```diff
@@ -55,6 +56,11 @@ def __init__(
         drop_rate=0.1,
         original_compatible=False,
         sampling_rate=100,
+        highpass_axis=None,
```

Comment:
For DIY principles, I think we should rather put this in base.py. I'll take care of this.

Reply:
Fixed.